packages/cli/src/commands/mkguest.js

```diff
@@ -3,8 +3,8 @@ import os from 'os';
 import { E } from '@endo/far';
 import { withEndoParty } from '../context.js';
 
-export const mkguest = async ({ name, partyNames }) =>
+export const mkguest = async ({ name, partyNames, introducedNames }) =>
   withEndoParty(partyNames, { os, process }, async ({ party }) => {
-    const newGuest = await E(party).provideGuest(name);
+    const newGuest = await E(party).provideGuest(name, introducedNames);
```

Comment:
This seems fine. Elsewhere, I’ve used parallel arrays. Parallel arrays might adapt better if we allow pet name paths on the key side.

Reply:
should this be an options bag?

Reply:
It’s worth considering. None of these APIs are future-proofed yet.